pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -19,9 +30,53 @@ workflow RNAWithUMIsPipeline {
 		File exonBedFile
 	}
 
+    if (defined(bam) && (defined(r1_fastq) || defined(r2_fastq))) {
+      call utils.ErrorWithMessage as ErrorMessageDoubleInput {
+        input:
+          message = "Bam and fastq files cannot both be defined as input"
+      }
+    }
+
+    if (!defined(bam) && (!defined(r1_fastq) && !defined(r2_fastq))) {
+      call utils.ErrorWithMessage as ErrorMessageMissingInput {
+         input:
+           message = "Either bam or fastqs must be defined."
+      }
+    }
+
+    if (defined(r1_fastq) || defined(r2_fastq)) {
+      if ((defined(r1_fastq) && !defined(r2_fastq)) || (defined(r2_fastq) && !defined(r1_fastq))) {
+        call utils.ErrorWithMessage as ErrorMessageMissingR2Fastq {
+          input:
+            message = "R1_fastq and r2_fastq must both be defined"
+        }
+      }
+      if (!defined(library_name) || (!defined(platform)) || (!defined(platform_unit)) || (!defined(read_group_name)) || (!defined(sequencing_center))) {
+        call utils.ErrorWithMessage as ErrorMessageMissingFastqHeaderMetadata {
+          input:
+            message = "If r1_fastq and r2_fastq are defined then library_name, platform, platform_unit, read_group_name, and sequencing center must also be defined"
+        }
+      }
```

Comment:
May make sense to pull this out to its own task and just document the reasoning behind fastq vs ubam inputs